packages/app/src/messages/constants/event.enum.ts

```diff
@@ -10,6 +10,7 @@ export enum MessagesEvent {
   AddNewMessage = "ADD_NEW_MESSAGE",
   ClearAllThreads = "CLEAR_ALL_THREADS",
   HideDeleteModal = "HIDE_DELETE_MODAL",
+  DeleteTemplates = "DELETE_TEMPLATES",
```

Comment:
this line is redundant

Reply:
Removed in [ca0ccd5](https://github.com/mudita/mudita-center/pull/752/commits/ca0ccd5977773430798c01ef71f22f8086da8e6b)